webapp/src/ts/modules/reports/reports-content.component.ts

```diff
@@ -172,6 +172,9 @@ export class ReportsContentComponent implements OnInit, OnDestroy {
   }
 
   search(query) {
+    if (this.selectMode) {
```

Comment:
This is kinda weird in mobile. 
There is no visual difference between being in select mode or not when seeing report detail, but if I click on the case_id while in select mode, then the link doesn't do anything. 
I think we should be consistent somehow?

Reply:
I brought up the topic to last week's Care Teams meeting. We're going to review the experience when everyone from the trio is back from holidays. There's a [ticket](https://github.com/medic/cht-core/issues/8419) for tracking. 
For now, we're just going to make the filter work, i.e. fix the bug. We'll fix the experience after evaluation.
For now, if the select mode is on, the link will be gray. 
<img width="717" alt="Screenshot 2023-07-31 at 11 23 28 am" src="https://github.com/medic/cht-core/assets/66472237/fcd6b1de-6898-4d79-8dba-6d4803eb1f24">